src/main/java/org/openstreetmap/atlas/geography/atlas/delta/AtlasDeltaGenerator.java

```diff
@@ -1,30 +1,31 @@
 package org.openstreetmap.atlas.geography.atlas.delta;
 
-import org.openstreetmap.atlas.exception.CoreException;
-import org.openstreetmap.atlas.geography.atlas.AbstractAtlas;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
 import org.openstreetmap.atlas.geography.atlas.Atlas;
 import org.openstreetmap.atlas.geography.atlas.AtlasResourceLoader;
 import org.openstreetmap.atlas.streaming.resource.File;
 import org.openstreetmap.atlas.streaming.resource.FileSuffix;
-import org.openstreetmap.atlas.utilities.conversion.StringConverter;
 import org.openstreetmap.atlas.utilities.runtime.Command;
 import org.openstreetmap.atlas.utilities.runtime.CommandMap;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * @author matthieun
+ * @author hallahan
  */
 public class AtlasDeltaGenerator extends Command
 {
-    private static final Switch<String> BASE_SWITCH = new Switch<>("base",
-            "The name of the base Atlas", StringConverter.IDENTITY, Optionality.REQUIRED);
+    private static final Switch<Path> BEFORE_SWITCH = new Switch<>("before",
+            "The before atlas from which to delta.", Paths::get, Optionality.REQUIRED);
 
-    private static final Switch<String> ALTER_SWITCH = new Switch<>("alter",
-            "The name of the alter Atlas", StringConverter.IDENTITY, Optionality.REQUIRED);
+    private static final Switch<Path> AFTER_SWITCH = new Switch<>("after",
+            "The after atlas that the before atlas deltas to.", Paths::get, Optionality.REQUIRED);
 
-    private static final Switch<File> OUTPUT_FOLDER_SWITCH = new Switch<>("outputFolder",
-            "The path of the output folder", string -> new File(string), Optionality.REQUIRED);
+    private static final Switch<Path> OUTPUT_DIR_SWITCH = new Switch<>("outputDir",
+            "The path of the output directory.", Paths::get, Optionality.REQUIRED);
```

Comment:
Nit: outputDir -> outputDirectory. As @lucaspcram said we try to avoid abbreviations, even popular ones